catalog/dags/common/popularity/constants.py

```diff
@@ -0,0 +1,9 @@
+IMAGE_VIEW_NAME = "image_view"
+AUDIO_VIEW_NAME = "audio_view"
+AUDIOSET_VIEW_NAME = "audioset_view"
+IMAGE_POPULARITY_CONSTANTS_VIEW = "image_popularity_constants"
+AUDIO_POPULARITY_CONSTANTS_VIEW = "audio_popularity_constants"
+IMAGE_POPULARITY_PERCENTILE_FUNCTION = "image_popularity_percentile"
+AUDIO_POPULARITY_PERCENTILE_FUNCTION = "audio_popularity_percentile"
+STANDARDIZED_IMAGE_POPULARITY_FUNCTION = "standardized_image_popularity"
+STANDARDIZED_AUDIO_POPULARITY_FUNCTION = "standardized_audio_popularity"
```

Comment:
These were moved to avoid circular imports between`storage/columns.py` and `popularity/sql.py`.